### PR TITLE
Add Makefile

### PR DESCRIPTION
**Pull request by praveenkumar**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,43 @@
+# Go and compilation related variables
+BUILD_DIR ?= out
+
+ORG := github.com/code-ready
+REPOPATH ?= $(ORG)/goodhosts-cli
```
Comment (cfergeau) [inline Makefile]: ORG and REPOPATH do not seem to be used

File: Makefile [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+BINARY_NAME := crc-goodhosts-cli
+RELEASE_DIR ?= release
+
+vendor:
```
Comment (cfergeau) [inline Makefile:11]: I don't think we want `vendor` to be the target run by default when one runs `make`?
Need `all: xxx`, `install` would be the best choice given what we have, but I'd usually expect make to build for the current platform, and that's it, install would be a separate step.

**Review by cfergeau — COMMENTED** [review]
Would be good to have a link to the upstream PR

File: Makefile (hunk 2 above)
Comment (praveenkumar) [reply Makefile:11]: @cfergeau yes added the default target which is `install`

**Comment by praveenkumar** [comment]
> Would be good to have a link to the upstream PR
@cfergeau  https://github.com/goodhosts/cli/pull/4

File: Makefile [hunk 3]
```diff
@@ -0,0 +1,45 @@
+# Go and compilation related variables
+BUILD_DIR ?= out
+
+BINARY_NAME := goodhosts
+RELEASE_DIR ?= release
+
+# Add default target
+.PHONY: default
+default: install
```
Comment (cfergeau) [inline Makefile]: `all` is more usual than `default`

Comment (praveenkumar) [reply Makefile]: Done

**Pull request merged**